connectivity/builder/client_egress_expression.go

```diff
@@ -8,16 +8,33 @@ import (
 
 	"github.com/cilium/cilium-cli/connectivity/check"
 	"github.com/cilium/cilium-cli/connectivity/tests"
+	"github.com/cilium/cilium-cli/utils/features"
 )
 
 //go:embed manifests/client-egress-to-echo-expression.yaml
 var clientEgressToEchoExpressionPolicyYAML string
 
+//go:embed manifests/client-egress-to-echo-expression-port-range.yaml
+var clientEgressToEchoExpressionPolicyPortRangeYAML string
+
 type clientEgressExpression struct{}
 
 func (t clientEgressExpression) build(ct *check.ConnectivityTest, _ map[string]string) {
+	clientEgressExpressionTest(ct, false)
+	if ct.Features[features.PortRanges].Enabled {
+		clientEgressExpressionTest(ct, true)
+	}
```

Comment:
Is this running the same test twice, one with and without port ranges? I see that this pattern is applied to all the other tests that were modified. Wouldn't this in theory double the amount of time to run tests? If so, is this really necessary? Why not just run the test with port ranges enabled if it's available?

Reply:
Running these tests in production CI doesn't consume much additional time. I am concerned about removing simple port tests from the connectivity tests in the long term as it could lead us to miss a bug.